docker/setup.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
```

Comment:
this is good as a first version, but should this script take as inputs which git repo to clone and which framework we want to run in (and possible which dataset to setup)?

Reply:
added arguments piped through Dockerfile for framework, dataset and git repo